.github/workflows/e2e-test.yml

```diff
@@ -41,9 +46,50 @@ jobs:
 
       - name: generate local SSH key
         run: ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa -N ""
+      
+      - name: Install Python deps
+        run: pip3 install requests wheel boto3
+
+      - name: Install go-junit-report
+        run: go install github.com/jstemmer/go-junit-report/v2@latest
 
       - name: run e2e test
-        run: make e2e
+        run: |
+          status=0
+          if ! make e2e > test_output.txt; then
```

Comment:
Can we directly print the output without writing it to a file? I think the file is not necessary as it will never be used somewhere else?